content/en/blog/2019/introducing-istio-operator/index.md

```diff
@@ -9,29 +9,28 @@ target_release: 1.4
 ---
 
 Kubernetes [operators](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/) provide
-a pattern for encoding human operations knowledge in software and are a popular way to simplify
-the administration of software infrastructure components.
-Because Istio is complex to administer, it's a natural candidate for an operator.
-
-Up until now, [Helm](https://github.com/helm/helm) has been the primary tool to install and upgrade Istio.
-This approach worked well but had some disadvantages:
-
-1. Users need to install another tool.
-1. Supporting Istio's many small specializations cause Helm templates to become unwieldy.
-1. Helm configurations are difficult to validate using Istio's proto-based APIs.
-1. Upgrades sometimes require Istio specific hooks that have been difficult to implement and maintain with Helm.
-
-Starting with Istio 1.4, the [Helm installation](/docs/setup/install/helm/) approach has been deprecated
-in favor of a new [installation using {{< istioctl >}}](/docs/setup/install/istioctl/) approach.
-Upgrading from Istio 1.4 onward (that is, versions not initially installed with Helm)
-will also be done using a new [{{< istioctl >}} upgrade feature](/docs/setup/upgrade/istioctl-upgrade/).
-
-The new `istioctl` commands use a
-[Custom Resource Definition (CRD)](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions)
-to configure the installation.
-The CRD is part of a new Istio operator implementation intended to simplify the common administrative tasks of
-installation, upgrade, and complex configuration changes for Istio.
-Validation and checking for installation and upgrade is tightly integrated with the tools to prevent
+a pattern for encoding human operational knowledge in software and are a popular way to simplify
+the administration of software infrastructure components. Istio is aa natural candidate for an automated
+operator as it is challenging to administer.
+
+[Helm](https://github.com/helm/helm) has been the primary tool to install and upgrade Istio. This approach
+worked well but had some disadvantages:
```

Comment:
I like @ssuchter 's proposal to invert the disadvantages to advantages:

[Helm](https://github.com/helm/helm) has been the primary tool to install and upgrade Istio. The new istioctl based installation has several advantages:

- Users just install one tool, istioctl
- All API fields are validated
- Small customizations not in the API do not require chart or API changes
- Version specific upgrade hooks can be easily and robustly implemented

Reply:
addressed.